src/quo/__visual_tests__/button.e2e.js

```diff
@@ -29,7 +29,7 @@ const loginToHomePage = async () => {
   await waitToNavigate(200);
   await element(by.id('browser-stack')).tap();
   await waitToNavigate(400);
-  await element(by.text('Quo2.0 Preview')).tap();
+  await element(by.text('Quo Preview')).tap();
```

Comment:
Does anybody use these *visual tests*? Shouldn't we remove `__visual_tests__` entirely?

Reply:
this should be removed for now yes, it was missed on the previous clean up

Reply:
oh cool , removing __visual_tests__ entirely

Reply:
done